languages/react-native/example/ios/Podfile

```
@@ -0,0 +1,43 @@
+
+  # Run Codegen during development for the example app.
+  pre_install do |installer|
+    system("cd ../../ && npx bob build --target codegen")
+  end
+
+  post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+    react_native_post_install(
+      installer,
+      config[:reactNativePath],
+      :mac_catalyst_enabled => false,
```

RuboCop (1.75.5)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)